composer.json

```diff
@@ -7,9 +7,9 @@
     "require-dev": {
```

Comment:
We can discuss. I generally rely on semver unless a library has proven to not do it properly. New patches are desired and generally, I like getting the latest additions too.